internal/analytics/events.go

```diff
@@ -42,8 +45,36 @@ func MockedTracker(name string, action string, flags []string) *MockTracker {
 			"id":      id,
 			"name":    name,
 		},
-	}
-	tracker := MockTracker{ID: id}
-	tracker.On("SendEvent", mockedTrackingArgs...)
+	}...)
+	tracker.On("SendEvent", []interface{}{
+		"testAccessToken",
+		"http://test.com",
+		"CLI Command Completed",
+		map[string]interface{}{
+			"id":      id,
+			"outcome": outcome,
+		},
+	}...)
 	return &tracker
 }
+
+func SendCommandRunEvent(name string, cmd *cobra.Command, analyticsTracker Tracker) {
```

Comment:
How about making these methods on the tracker instead of functions that take a tracker?
```
func (a analyticsTracker) SendCommandRunEvent(accessToken string, baseURI string, eventName string, properties map[string]interface{}) {}
func (a analyticsTracker) SendCommandCompletedEvent(accessToken string, baseURI string, outcome string) {}
```